Formula/libcaca.rb

```diff
@@ -1,20 +1,14 @@
 class Libcaca < Formula
   desc "Convert pixel information into colored ASCII art"
   homepage "http://caca.zoy.org/wiki/libcaca"
-  url "http://caca.zoy.org/raw-attachment/wiki/libcaca/libcaca-0.99.beta19.tar.gz"
-  mirror "https://www.mirrorservice.org/sites/distfiles.macports.org/libcaca/libcaca-0.99.beta19.tar.gz"
-  mirror "https://fossies.org/linux/privat/libcaca-0.99.beta19.tar.gz"
-  version "0.99b19"
-  sha256 "128b467c4ed03264c187405172a4e83049342cc8cc2f655f53a2d0ee9d3772f4"
+  url "https://github.com/cacalabs/libcaca/releases/download/v0.99.beta20/libcaca-0.99.beta20.tar.bz2"
+  mirror "https://fossies.org/linux/privat/libcaca-0.99.beta20.tar.bz2"
+  version "0.99b20"
+  sha256 "ff9aa641af180a59acedc7fc9e663543fb397ff758b5122093158fd628125ac1"
   license "WTFPL"
-  revision 3
 
-  # The regex here is looser than usual because it has to match unstable
-  # versions for now. If this software is updated in the future to a stable
-  # version, this regex should be modified to not match unstable versions.
   livecheck do
-    url :homepage
-    regex(/href=.*?libcaca[._-]v?(\d+(?:\.\d+)+.*?)\.t/i)
+    url :stable
   end
```

Comment:
Since pointing to github, this block can be removed?

Reply:
Without the livecheck block:
```
$ brew livecheck -d libcaca
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/libcaca.rb

Formula:          libcaca
Livecheckable?:   No

URL:              https://github.com/cacalabs/libcaca/releases/download/v0.99.beta20/libcaca-0.99.beta20.tar.bz2
URL (processed):  https://github.com/cacalabs/libcaca.git
Strategy:         Git

URL:              https://fossies.org/linux/privat/libcaca-0.99.beta20.tar.bz2
Strategy:         None

URL:              https://github.com/cacalabs/libcaca.git
Strategy:         Git

URL:              http://caca.zoy.org/wiki/libcaca
Strategy:         None
Error: libcaca: Unable to get versions
```
With:
```
$ brew livecheck -d libcaca
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/libcaca.rb

Formula:          libcaca
Livecheckable?:   Yes

URL (stable):     https://github.com/cacalabs/libcaca/releases/download/v0.99.beta20/libcaca-0.99.beta20.tar.bz2
URL (processed):  https://github.com/cacalabs/libcaca.git
Strategy:         Git

Matched Versions:
0.99.beta14, 0.99.beta15, 0.99.beta16, 0.99.beta17, 0.99.beta18, 0.99.beta19, 0.99.beta20
libcaca : 0.99b20 ==> 0.99.beta20
```

Reply:
I see, looks good then. 👍